src/shared/components/BlockList/BlockList.tsx

```diff
@@ -23,7 +23,7 @@ const BlockList: FC<BlockListProps> = ({ blocks, config }) => {
 		const layout = (children?: ReactNode, backgroundColor?: string) => (
 			<div
 				className="u-p-0 c-block-list__item"
-				style={{ backgroundColor }}
+				style={{ backgroundColor, '--block-background': backgroundColor } as CSSProperties}
```

Comment:
optional: add type for styles that contain css variables:
https://stackoverflow.com/a/70398145/373207

Reply:
https://github.com/viaacode/avo2-client/pull/2005/commits/97b749ac6735014589904accccd9431e7531792e